classes/tasks/OpenAccessNotification.inc.php

```diff
@@ -113,11 +65,20 @@ public function sendNotifications($journal, $curDate)
                     $users = Repo::user()->getMany(
                         Repo::user()->getCollector()
                             ->filterByContextIds([$journal->getId()])
-                            ->filterBySettings(['openAccessNotification' => 1])
+                            //->filterBySettings(['openAccessNotification' => 1])
```

Comment:
Does anything need to happen with this comment?

Reply:
Thanks, forgot to resolve that. I think this setting was present in some old versions of OJS but doesn't exist anymore. I'd prefer to leave this as it was originally until open access notifications get refactored (or removed).

Reply:
I'm worried that this is going to be sending notifications to every use in the system. If something is broken or semi-broken, I'd prefer to leave it in a safer state.

@asmecher you've worked most with the subscriptions stuff. Can you make a judgement call on what to do regarding this open access notification? Is there a simple solution to figure out who should receive this notification? Or should we disable it entirely if it's not working?

Reply:
At this point I think it would be best rewritten fully when it's prioritized. I [recommended removing it in 2021](https://github.com/pkp/pkp-lib/issues/490#issuecomment-763000360) and still think that's best. If you'd rather leave it in place, please keep it safe by leaving the `openAccessNotification` setting check in.